apis/users.py

```diff
@@ -330,6 +363,27 @@ def confirm_email():
     return "Email Confirmed"
 
 
+@user_api.post("/check_email_confirmation")
+def check_email_confirmation():
+    username = request.json
+
+    # user must be logged in to access
+    if session["username"] != username:
+        return "false"
```

Comment:
Give more meaningful responses here and below (i.e. inform the user the email is already confirmed), also make use of HTTP status codes (i.e. 400 when not logged in).

Reply:
Fixed